src/ddspdrum/module.py

```diff
@@ -17,9 +19,11 @@ class SynthModule:
     Base class for synthesis modules.
     """
 
-    def __init__(self):
-        self.sample_rate = SAMPLE_RATE
-        self.control_rate = CONTROL_RATE
```

Comment:
I'm still really not sure why you want to make your life so complicated. But if it makes you happy I'm fine with it.

Reply:
Global variables are easy ways to shoot yourself in the foot down the road, and to create hard to find bug.

Reply:
Cool.